devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -74,11 +74,11 @@ metadata {
 def installed() {
 	if (zwaveInfo?.model.equals("0051")) {
 		state.numberOfSwitches = 2
-	} else {
+	} else if (zwaveInfo?.model.equals("0052") || zwaveInfo?.model.equals("0053")) {
 		state.numberOfSwitches = 1
```

Comment:
you'd still need to add an explicit else here I think to avoid an NPE (just in case)

Also this doesn't really alter any of the behavior of the device, since these two models would have been defaulting to 1 in the first place, right?

Reply:
Okay. Please add back a default case here (or remove the model checks you added).

Reply:
@MGoralczykS